src/System.IO/tests/System.IO.Tests.csproj

```diff
@@ -59,6 +59,12 @@
     <Compile Include="$(CommonTestPath)\System\IO\WrappedMemoryStream.cs">
       <Link>Common\System\IO\WrappedMemoryStream.cs</Link>
     </Compile>
+    <Compile Include="TextReader\CharArrayTextReader.cs" />
+    <Compile Include="TextReader\TextReaderTests.cs" />
+    <Compile Include="TextReader\TextReaderTests.netcoreapp..cs" Condition="'$(TargetGroup)' == 'netcoreapp'" />
```

Comment:
do yo uhave an extra period in the file name here? `..cs`

Reply:
oops...